### DISCUSSION TITLE
AssetSource has prefix 'assets', can I remove it?

### DISCUSSION THREAD

**Discussion by CalsRanna** [bluefireteam/audioplayers#1530] [General]
I don't want the default prefix of assets, how can I remove or replace it?

### Affected platforms

Linux

**Comment by Gustl22** [comment]
You can set the default path by overriding the [AudioCache](https://github.com/bluefireteam/audioplayers/blob/main/getting_started.md#audiocache) and [set another prefix](https://github.com/bluefireteam/audioplayers/blob/415dda3b1621c57ea4b0366187f27f6a189555bf/packages/audioplayers/lib/src/audio_cache.dart#LL51C1-L51C1).
